### ISSUE TITLE
想换成densenet的前端

### ISSUE THREAD

**Issue by Kenneth-X** [mowayao/gluon_SSD#2]
cur = 0
		i = 0
		while i < len(cfg):
			v = cfg[i]
			if v == 'M':
				with self.name_scope():
					self.vgg.add(nn.MaxPool2D(pool_size=2, strides=2))
				i += 1
			elif v == 'C':
				with self.name_scope():
					self.vgg.add(nn.MaxPool2D(pool_size=2, strides=2, ceil_mode=True))
				i += 1
			elif isinstance(pretrained_layers[cur], nn.Conv2D):
				#conv2d = pretrained_layers[cur]
				with self.name_scope():
					self.vgg.add(nn.Conv2D(v, 3, padding=1))
					self.vgg.add(nn.Activation('relu'))
				i += 1
			cur += 1

这部分的代码怎么处理“”？“？”？
